client-side/src/stories/DateTime/DateInputs.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import PropTypes from 'prop-types';
+
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
+import { DateTimeRangePicker } from '@mui/x-date-pickers-pro/DateTimeRangePicker';
+
+import './DateInputs.scss';
+
+const DateInputs = ({onAccept=undefined})=>{
```

Comment:
the onAccept variable is defined as required so you don't need the undefined default value